packages/node.resolver/src/index.ts

```diff
@@ -24,6 +24,25 @@ export async function resolveNodeVersion (
   return pickedVersion.substring(1)
 }
 
+export async function resolveNodeVersionList (
```

Comment:
This function duplicates a lot of code from resolveNodeVersion. Try to refactor it.

Reply:
Thanks, I will try it